src/backend/index.ts

```diff
@@ -32,3 +34,9 @@ app.get("/api/author", (req: express.Request, res: express.Response) => {
 app.get("/api/pr", (req: express.Request, res: express.Response) => {
   prGet(req, res);
 });
+app.get("/health", (req: express.Request, res: express.Response) => {
+  health(req, res);
+});
+app.get("/version", (req: express.Request, res: express.Response) => {
+  version(req, res);
```

Comment:
i do not understand why backend call and not prefix with `/api` because this is against the design that we have.

Reply:
This endpoint is expected to be used by users as a page to test their configuration, and not as an API. They want to be able to visit it on their browser. Pre-fixing it with api/ would mean it should be used as a REST API only?

Reply:
on a pure design standpoint, we want to distinguish between static html content and backend based dynamic content. What you are doing here break this design model. That's eventually (as we add more codes) we no longer can tell what are static and what are dynamic. on a the browser, user just have to enter /api/version to get the version if we prefix `/api`, please let me know if this does not work. IMO, we should have a modal dialog box to show the version and not showing version to customer in this manner.

Reply:
@dennisseah That will come in the future, we're tackling the feature in chunks. Building and designing the front end for this will be a long term solution while having an endpoint users can visit in the meantime is still helpful for debugging. If what I'm doing here is breaking the model, then we perhaps need to re-think the model because according to this model, it sounds like adding any page other than * for static content is not allowed :)

Reply:
I believe the URL convention is clear on when we use /api and when not. I am not clear about "model" that you mentioned here. Maybe I am missing some information. adding pages or not is independent of URL convention. IMO, we should not mixed them together.

Reply:
@samiyaakhtar update?

Reply:
@andrebriggs pending sync with @gemorris. the suggestions made here are going to need some rework